src/mgr/BaseMgrStandbyModule.cc

```diff
@@ -61,21 +61,41 @@ ceph_get_mgr_id(BaseMgrStandbyModule *self, PyObject *args)
 }
 
 static PyObject*
-ceph_config_get(BaseMgrStandbyModule *self, PyObject *args)
+ceph_get_module_option(BaseMgrStandbyModule *self, PyObject *args)
 {
   char *what = nullptr;
-  if (!PyArg_ParseTuple(args, "s:ceph_config_get", &what)) {
+  if (!PyArg_ParseTuple(args, "s:ceph_get_module_option", &what)) {
     derr << "Invalid args!" << dendl;
     return nullptr;
   }
 
   std::string value;
   bool found = self->this_module->get_config(what, &value);
   if (found) {
-    dout(10) << "ceph_config_get " << what << " found: " << value.c_str() << dendl;
+    dout(10) << __func__ << " " << what << " found: " << value.c_str() << dendl;
     return PyString_FromString(value.c_str());
   } else {
-    dout(4) << "ceph_config_get " << what << " not found " << dendl;
+    dout(4) << __func__ << " " << what << " not found " << dendl;
+    Py_RETURN_NONE;
+  }
+}
+
+static PyObject*
+ceph_option_get(BaseMgrStandbyModule *self, PyObject *args)
```

Comment:
Aside from the type of the self parameter, this looks like a duplicate of `ceph_option_get()` from BaseMgrModule.cc. It'd be kinda nice to unify the two, but given there's a bunch of other functions already with subtly different implementations between the two classes, that looks like a separate larger piece of potential cleanup work for another time :-)

Reply:
agree.. i'm going to leave that cleanup for later, though, as this pr is blocking some other stuff